hex-number/hex-number.py

```diff
@@ -0,0 +1,102 @@
+class Node:
```

Comment:
VERIFICATION RESULT

Your pull request: HEXNUMBER-1021 Added hex-number.py and test-hex-number.py


Your commit: HEXNUMBER-1021 Added hex-number.py

Your commit: HEXNUMBER-1021 Added test-hex-number.py